hack/verify-gofmt.sh

```diff
@@ -0,0 +1,20 @@
+
+#!/bin/bash
+
+function list_go_src_files() {
+	find . -not \( \
+		\( \
+		-wholename './.*' \
+		-o -wholename '*/vendor/*' \
+		\) -prune \
+	\) -name '*.go' | sort -u
+}
+
+
+bad_files=$(list_go_src_files | xargs gofmt -s -l)
```

Comment:
thinking about replace `gofmt` with `goimports`, WDYT?

Reply:
i think if we are going to use the `verify-diff.sh` script from #90, then we should centralize all the logic there.

re: `goimports`, i haven't used it before but it seem nice to me